mqtt/mqtt-broker/src/settings/mod.rs

```diff
@@ -153,3 +153,74 @@ pub struct SessionPersistenceConfig {
     time_interval: Duration,
     unsaved_message_count: u32,
 }
+
+#[derive(Debug, Clone, Deserialize)]
+pub struct Enableble<T> {
```

Comment:
typo? Enable<T>?

Reply:
Not exactly a typo. It is some type, that can be enabled/disabled. But, probably Enable also a good option.